remove spectral norm, add django migrate benchmark.

toshok opened the pull request:
switch back to python parser (-x) so django migrate benchmark works.

kinda large since I imported all of django from lib_pyston into the benchmark_suite dir.  should we instead assume it's already installed?

Commit:
remove spectral norm, add django migrate benchmark.  switch back to python parser (-x) so django migrate benchmark works.

Commit:
add django_migrate to the list of interp-only benchmarks, and use -x for interp-only as well

kmod commented:
sgtm, though maybe we should create a lib/ directory somewhere in here since I think we might end up adding a number of external libraries?  either at the same level as benchmark_suite or inside it.

toshok commented:
nod.  that sounds like a better solution than polluting benchmark_suite

On Mon, Mar 30, 2015 at 3:42 PM, Kevin Modzelewski <notifications@github.com

> wrote:
> 
> sgtm, though maybe we should create a lib/ directory somewhere in here
> since I think we might end up adding a number of external libraries? either
> at the same level as benchmark_suite or inside it.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/dropbox/pyston-perf/pull/3#issuecomment-87862733.

Commit:
move some things around to clean up the django benchmark

- move the benchmark_suite/django/ dir into benchmark_suite/lib/
- rename benchmark_suite/testsite/ to benchmark_suite/django_migrate_testsite/
